### PR TITLE
Create Account Integration

### PR DESCRIPTION
**Pull request by hornilla-jowelyn**
- created service
- created transformer utils
- submit create form to create multiclique account api

[MultiClique---Create-Account.webm](https://github.com/deep-ink-ventures/multiclique-frontend/assets/137761725/3b1fd590-04cf-4418-88af-15bff0ddc096)

### CODE REVIEW

File: src/components/SignatoriesForm.tsx [hunk 1]
```diff
@@ -83,15 +83,15 @@ const SignatoriesForm = (props: {
                   placeholder='Wallet Address'
                   className='input input-primary'
                   disabled={disabled}
-                  {...register(`${props.formName}.${index}.walletAddress`, {
+                  {...register(`${props.formName}.${index}.address`, {
```
Comment (GHesericsu) [inline src/components/SignatoriesForm.tsx:86]: Could you change line 43 too please.

File: src/pages/account/create.tsx [hunk 2]
```diff
@@ -47,6 +53,20 @@ const Create = () => {
     } catch (err) {
       handleErrors('Error in transferring ownership to multisig', err);
     }
+
```
Comment (GHesericsu) [inline src/pages/account/create.tsx:56]: Could you get rid of all the code on this file here because I will do this part? :)

**Review by GHesericsu — CHANGES_REQUESTED** [review]
Just two changes please

File: src/components/SignatoriesForm.tsx (hunk 1 above)
Comment (hornilla-jowelyn) [reply src/components/SignatoriesForm.tsx:86]: Updated field name in `append`

File: src/pages/account/create.tsx (hunk 2 above)
Comment (hornilla-jowelyn) [reply src/pages/account/create.tsx:56]: This is now removed

**Pull request merged**